spec/foxsi2_count_spectrum.pro

```diff
@@ -65,19 +68,39 @@ area = get_foxsi_effarea( $
 		energy=emid, module_num = module_num, use_theoretical=use_theoretical, $
 		offaxis_angle=offaxis, _extra=_extra )
 
-if keyword_set(stop) then stop
+;if keyword_set(stop) then stop
 
 ; fold flux with FOXSI response
+eff_fudge = (1-0.05)
 counts = flux*area.eff_area_cm2  ; now the units are counts per second per keV
 
 ; Smear to account for imperfect energy resolution (~0.5 keV).
 if keyword_set(smear) then begin
 	i=where( emid gt 3. and emid lt 30. )
 	nbins = 0.5 / average(ewid)
 	ni = n_elements(i)
-	s = smooth( flux[i], nbins )
-	if i[ni-1] gt (n_elements(flux)-1) then flux2 = [flux[0:i[0]-1],s] $
-		else flux2 = [flux[0:i[0]-1],s,flux[ni+1:n_elements(flux)-1]]
+	s = smooth( counts[i], nbins )
+	;PSA added  the following to get the flux to counts with smear
+	if i[ni-1] ge (n_elements(counts)-1) then ncounts = [counts[0:i[0]-1],s] $
+		else counts2 = [counts[0:i[0]-1],s,counts[ni+1:n_elements(counts)-1]]
```

Comment:
Should this also be called ncounts? "counts2" doesn't appear again in the function.